### ISSUE TITLE
KRE 3.0 Providing incentives to create demand for all participants of the ecosystem

### ISSUE THREAD

**Issue by Raketenernie** [kinecosystem/technology#7]
[KRE 3.0.pdf](https://github.com/kinecosystem/technology/files/4785806/KRE.3.0.pdf)

**Comment by Polype01** [comment]
Hello Raketenernie,


We have hyper inflation because we do not have a real demand/speculative demand for KIN and a lot of KIN distributed through the KRE.
We do not have a real demand because we do not have valid SPENDS opportunities for existing Apps and because not so many new partners are joining the Ecosystem currently
We do not have a real demand for KIN because of the SEC v. Kik case

So just because KIN has a low value now, we have to switch the KIN KRE logic and provide dollars instead of KIN? Like in all other weak economies from third-world countries, you suggest to leave aside the official KIN Ecosystem currency and use dollars like in all black markets. Is this what you are proposing? How would your proposal would ensure that KIN would thrive after this?
Is spending US dollars in Venezuela helping in increasing the value of the Venezualian Bolivar and sustaining the economy?

Tourism is low because there is no incentive to move the KIN from one app to another app as there are no real, big interesting and varied spends for the moment.
Tourism is low because I don't have a reason to move my KIN from App 1 to App2. 
To me, it is the same, all things being equal, and it is not worth the effort to move the KIN between Apps. Low KIN price is NOT the reason why I choose not to move my KIN between Apps. It is becuase I have no incentive to do it

Reducing supply to increase demand. I get your point but KF is not going to burn coins/reduce the payout to Apps (risks that the bigger partners would leave)
KF might not even be able to reduce the KIN payments as it might be contractual disclosures that we do not know. 

You propose distributing dollars instead and no KIN for a period. OK. It seems not sustanaible to me. WHeRe WoUlD thE mOnEy ComE fRoM, tHe SKy?  Once the 2.4M from Solana will be finished, how can you ensure long term that this will continue? You do not know yourself how this will be after that period.

Also, if KIN has no value now according to you, how will giving dollars instead of KIN help to increase the value of KIN itself? Partners will get more value (dollars) but how will KIN get a higher value out of it? Do you think that all partners will reinvest their dollars into the KIN Ecosystem and buy KIN? We are not sure. They might as well get as much as dollars as they need and just quit the Ecosystem.

What is the added value to distribute dollars to Ecosystem partners? Will it serve the adoption and development of KIN overall? Do you think Partners/entities will join just because KRE will distribute Fiat in place of KIN? Maybe. Aren't there other projects with a higher price that would be more suitable?

I fully agree with you that we have to raise demand. I believe this will be done by adapting many parameters and smoothly and organically and not only by cutting the supply abruptly and by swapping dollars instead of KIN.

KIN to be swapped for USD just to make an artificial reduction of KIN supply (and against the KIN philosophy) and having no insurance that it will have an effect on the Ecosystem development seems not good to me.

How can this be automated via the KIN KRE algorithm? Is your idea only to get an artificial raise of KIN price ?

**Comment by ghost** [comment]
@Polype01 This is how you discourage people from contributing. Please delete and try to re-word in a way that isn't condescending.

**Comment by Raketenernie** [comment]
I reply in bold to your comments:

1. We do not have a real demand because we do not have valid SPENDS opportunities for existing Apps and because not so many new partners are joining the Ecosystem currently
We do not have a real demand for KIN because of the SEC v. Kik case.
**This is partly true but you just quoted the syntomes as a result of the hyperinflation itself, descriping what is lacking does not focus on the core problem nor solves it the core problem**

2. So just because KIN has a low value now, we have to switch the KIN KRE logic and provide dollars instead of KIN? **Again you so hooked up on the syntome of a low value of kin, we have to tacle the problem why kin has a low value and yes mainly it is because of the free inflow of KIN. This is matter of fact, the statisitics show this fact**
3. Partners will get more value (dollars) but how will KIN get a higher value out of it? **By stoping giving them KIN they have a ngeative delta Supply vs Demand, meaning every month their supply of KIN will decrease until eventually they either need to refill their supply by buying it of exchanges or by decreasing the amount of kin giving to users for their earn and spends. Alone by these two factors you increase the value of kin externally on exchanges and internally within the apps. On top changing to the 3 variables I mentioned such as fiat spend for kin, you truley rewards those apps who bring real demand to the table.**
4. How would your proposal would ensure that KIN would thrive after this?
Is spending US dollars in Venezuela helping in increasing the value of the Venezualian Bolivar and sustaining the economy? **You do realize all the countries who used dollarization are examples of failed economies, I underline for you failed!!! The only way they were able to continue was to introduce dollars. I cannot ensure that KIN will thrive after this as the duration period or the amount might be to small, but what I definetly can say my approach will focus truley on the demand side and will bring back value to kin. I repeat once we have an upgoing circle it has positve external effects acting as a catalyst for the ecosystem, bring back speculators, bringing new apps and users**
5. Tourism is low because there is no incentive to move the KIN from one app to another app as there are no real, big interesting and varied spends for the moment.
Tourism is low because I don't have a reason to move my KIN from App 1 to App2.
To me, it is the same, all things being equal, and it is not worth the effort to move the KIN between Apps. Low KIN price is NOT the reason why I choose not to move my KIN between Apps. It is becuase I have no incentive to do it **You basically spoted that kin is not a true medium of exchange , which you perfectly explained not even in its very own ecosystem, this however would immediately change if had a constant growing value , than beyond the ecosystem it could be used outside in the real world for example for a coffee or a gift card etc... Would you not agree if you had an app and you know every month the kin I hold is worth more, would you not create as much spends as possible in order to get the max of kin inflow as possible, knowing that the kin will be much more  worth the next month. If you have buisness sense and want a return then you would do it**
6. Reducing supply to increase demand. I get your point but KF is not going to burn coins/reduce the payout to Apps (risks that the bigger partners would leave)
KF might not even be able to reduce the KIN payments as it might be contractual disclosures that we do not know. **Wrong please read again, I never asked in a single line for a supply reduction or a coin burn, I asked for stop KIN payouts wotsoever and convert them into dollars. Stop spreading false information**
7.Do you think that all partners will reinvest their dollars into the KIN Ecosystem and buy KIN? We are not sure. They might as well get as much as dollars as they need and just quit the Ecosystem.
**Its an equation and eventually they have to reinvest their dollars and buy KIN as they run dry. The point is not only to switch the KRE to dollars but only rewards those who push demand with fiat spend for kin. The so created positiv delta will eventually catch up with the supply and will force the apps to reinvest, which then starts the upward trend in terms of a sustainable increase of kin value and demand. You are right they could aquire as much dollars as possible and exit, but to be fair they can do similar wtih kin aquire as much kin as possible and exit. And to be fair some apps have done so by not even have switched to the kin blockchain. Would you not agree the incentive for an app is greater to stay within the ecosystem and be pro active rewarding them somehing of value instead of  an asset which has only 0,13% of its ATH value left. You have to think also a step ahead that as more they focus on the spends and more the denand increases the more the value of kin rises meaning ultimately the econimical benefit using kin will increase for each app, if you want so the econimical benefit of each app using kin right now is only 0.13% what is was once before.**
8. What is the added value to distribute dollars to Ecosystem partners? Will it serve the adoption and development of KIN overall? Do you think Partners/entities will join just because KRE will distribute Fiat in place of KIN? Maybe. Aren't there other projects with a higher price that would be more suitable? **Yes I certainly believe it will help to create more adoption and kin awareness. It is not crazy to believe that distributing FIAT could bring new apps to the ecosystem. Especially new apps are the one who need to rebuy KIN which would you instanly create a function economical circle. I cannot speak about other projects who might be more suitable for whatever reason. May be others reward more, but KIN does have a funtiong network with cross joined apps , which is appealing to medium seized apps. 
9. KIN to be swapped f**or USD just to make an artificial reduction of KIN supply (and against the KIN philosophy) and having no insurance that it will have an effect on the Ecosystem development seems not good to me. **Wrong it is not a reduction it is setting the supply fixed for now until we meet equilibrium**
10. I fully agree with you that we have to raise demand. I believe this will be done by adapting many parameters and smoothly and organically and not only by cutting the supply abruptly and by swapping dollars instead of KIN. **This is your personal view based on what facts? I provided you a concept both proven by history and econimcs. Demand will never ever happen in the current state, as again the incentive to buy , hold or earn kin is so little, as a direct result of the hyperinflation. the only way of stoping hyperinflation is puting pressure on the supply and increase the value of kin which then will bring the above asked demand combined with your oganical growth the ecosystem will eventually come to an equlibrium where rewards as such wont be neccessary.**
11. How can this be automated via the KIN KRE algorithm? Is your idea only to get an artificial raise of KIN price ? **As said in the proposal per month we reward 200.000 dollars distributed to apps accordingly to their share of the 3 presented variables where fiat spend for kin is the most important. It is a subzidized ecosystem focusing only on the demand leading to an increase of value of kin. It is not artificial as the increase of value comes from within the ecosystem from its very own demand. Combined with a change of incentive to hold kin , user, speculator and devs will increase their demand to acquire a portion x of KIN. It is exactly the oposite what the KRE does now , instead of subzidizing liquidity we subsidize true demand.** 

**I repeat it is not about the price of kin, it is about abolishing hyperinflation and rewarding demand and increasing demand and yes this means kins price should not be 0,13% what it used to be. 
I bet all my kin, if KIN would be close to ATH price levels, the ecosystem and the adoption of kin would be at least 10x bigger. Each app, user, speculator would have had a 93x higher incentive in terms of value to hold buy or use kin, just think about 93x times. What do you or anyone here do not understand , it is plane simple right there.**

**Comment by Throwawayburros** [comment]
> Wrong please read again, I never asked in a single line for a supply reduction or a coin burn, I asked for stop KIN payouts wotsoever and convert them into dollars. 

Where will the dollars come from?  If KF has to sell Kin to make dollars then nothing changes, as the top KRE earners are regularly selling Kin for dollars.  The KF is not block.one, they do not have near limitless funds to pay out dollars.

> Again you so hooked up on the syntome of a low value of kin, we have to tacle the problem why kin has a low value and yes mainly it is because of the free inflow of KIN. This is matter of fact, the statisitics show this fact

Its universally agreed that the current low price of Kin, is due to the threat of the SEC case.  Having a high liquidity of Kin on exchanges is nowhere near as damaging as the SEC case.

June 3rd 2019 Kin = $0.00003555
June 4th 2019 SEC news becomes public
June 5th 2019 Kin = $0.00002585
June 17th, 2020 Kin = $0.00000570

Price was slashed by ~33% because of the SEC news.  To put that in perspective, on March 12th 2020 the stock markets had the largest sell off in decades.  It was known as black Thursday.  Bitcoin lost 47% and the rest of crypto had higher percentage losses.    

You keep saying that the supply is too high for Kin.  But if we take the total supply of Ripple (XRP) and multiply it by 100, we get the same amount of tokens that Kin has of 10 trillion.  Current price of Ripple $0.192551, after changing it to Kins supply Ripple is now worth $0.00192551.  Its still 300x over our current price.

There are many other fundamental issues with what you are claiming.

**Comment by Raketenernie** [comment]
> Where will the dollars come from? If KF has to sell Kin to make dollars then nothing changes, as the top KRE earners are regularly selling Kin for dollars. The KF is not block.one, they do not have near limitless funds to pay out dollars.

We know that KF will get money from migrating to Solana and we know that KF had a buy back program over 1 Million dollars , how much was spend I cannot say. On top they still have some money and some partners with money. So I do not think money will be a problem, unless you are KF employee and you do the financial accounting and have insight,  do you? You are right KRE earners regulary selling kin for dollars, but this applies for the top tier apps, which are not all playing a fair game.  It will be true introducing my model it wont stop them right away to keep selling kin, however as longer you do the reversal of the KRE switching to dollars they will a) either run out of kin or adjust their payouts in terms of earns and spends, for example an earn today gives you 1.000 Kin may be in a year you only get 10 kin, that is called deflation leading to an increased intrinisic value of kin regardless its price on a exchange. You say they got nearly unlimitless funds, this you and I cannot know as we do not have insight, this is pure speculation. Especially for the medium seized or smaller apps it is more the opposite they lack of equity and need any funds they can get, and these are the ones who do not have billions of kin sitting in their wallet, these are the ones who will bring immedate demand bk to the ecosystem as their initial supply of kin is faster used.


> 
> Its universally agreed that the current low price of Kin, is due to the threat of the SEC case. Having a high liquidity of Kin on exchanges is nowhere near as damaging as the SEC case.

Is it universal agreed or is this your opinion? Show me the source and the evidence that it is so? If not this is your personal opinon. I agree with you there is definetly a negatively correlation but to blame primarily the SEC for KINs bad performance in all kind of fields is far too simple. I would say it is more the combination of many variables. I can certainly say since the KRE started the trading pair of kin droped from an average of 0.0000030 ETH to 0.00000003 ETH, when the SEC announcement came KIN was actually at levels 0.00000040 eth and higher and dropped to arround 0.0000030 ETH which is a sharp drop. But the factor 10 drop alone was the KRE and this I know for sure as I day traded alot. I witness every KRE payout was followed by a dump on exchanges this only recently stopped as they destroyed the market and there are no big buy orders left. Of course kin has lost value vs the dollar cuz of the drop of BTC, but you dont buy kin vs dollars (not directly) you buy it vs BTC and ETH so you must look at theirs performance to get a clear picture what has happened. 
First of all this is KIN and not Ripple , two different blockchains two different approach, two different approaches. You comparing bannas with Apples, I used Ripple only as an example how you can get your invested money and that it is not wasted. 
Your calculation multiplying a supply by 100 and then devide the price by 100 does not work like. History and econimic models have shown that in a sceanrio like this the price would not drop by 100 it would accelerate as there would be capital fligh,t as ppl try to save their value and they would sell their Ripple as fast as possible. Followed by a price inflation for goods and services which you could buy with Ripple. This would start a downward circle accelerating with each time you increase the supply making the incentive to hold or buy Ripple to decrease to a minimum. This combined with a growing sell presure as everyone wants to save their value, the drop will be much bigger and you see similar default levles as kin. What I just explained would have happend, has happened with kin. It is  hyperinflation in its purest form having an oversupply kin not even a tiny bit matching its demand both from the ecosystem and speculative demand. The SEC just made the price erode even further or speed up things.  And yes we do have oversupply this is fact. This not my opinion you can see in the very own puplished graphs and statistics of KF. Either you accept it or not but it is there.  To tacle oversupply you either reduce the supply or stimulate the demand. This is actually what I am aiming, limiting supply while pushing demand, exactly what any economist will tell you to do. 




> 
> June 3rd 2019 Kin = $0.00003555
> June 4th 2019 SEC news becomes public
> June 5th 2019 Kin = $0.00002585
> June 17th, 2020 Kin = $0.00000570
> 
> Price was slashed by ~33% because of the SEC news. To put that in perspective, on March 12th 2020 the stock markets had the largest sell off in decades. It was known as black Thursday. Bitcoin lost 47% and the rest of crypto had higher percentage losses.
> 
> You keep saying that the supply is too high for Kin. But if we take the total supply of Ripple (XRP) and multiply it by 100, we get the same amount of tokens that Kin has of 10 trillion. Current price of Ripple $0.192551, after changing it to Kins supply Ripple is now worth $0.00192551. Its still 300x over our current price.
> 
> There are many other fundamental issues with what you are claiming.

**Comment by clitl83** [comment]
Hi Raketenernie, 
thank you for all the effort you put into your proposal. I have a few questions and I would be happy if you take a few minutes to address it. 
1. "You basically spotted that kin is not a true medium of exchange". Accepting that KIN is not a medium of exchange, would mean accepting KIN is not money for the digital world. I don't think we should accept it at all. But let's assume we change the KRE: How and when would you make the transition back to a KIN based KRE, because, in the end, a Dollarization is not the end goal if KIN shall be used globally, right?
2. "Would you not agree if you had an app and you know every month the kin I hold is worth more" Can you please explain again, how you would motivate the participants (normal users) in the economy to spend their KIN, when they think that their KIN will be worth more next month?
3. Assuming SEC case is over and we finally got clarity, what would happen with the price, if a lot of new participants influx the ecosystem (which we don't know)? Wouldn't it lead to extreme price spikes, which is also not as healthy as extreme inflation? So what are your ideas to avoid this situation (i totally get from a speculators point of view how nice this would be, but from the view of a FED/ECB maybe not :))
I am looking forward and thanks for your time.

**Comment by Polype01** [comment]
**1.	This is partly true but you just quoted the syntomes as a result of the hyperinflation itself, descriping what is lacking does not focus on the core problem nor solves it the core problem**
Don’t get me wrong. Anything that can increase demand is good for the project. I just doubt your approach is the correct way of doing it. KIN usage and spends from more users and users buying KIN massively will make the real demand increase gradually. This is the core problem. I don’t think that a short-term KRE paid in dollars would fix the systemic issue that we currently have (lack of real demand) but your contribution is at least an input to discuss demand

**2.	Again you so hooked up on the syntome of a low value of kin, we have to tacle the problem why kin has a low value and yes mainly it is because of the free inflow of KIN. This is matter of fact, the statisitics show this fact**
As you are also following KIN for a long time, the current low price of KIN is the consequence of multiple factors. Pantera Capital dumping Billions of KIN, crypto market crashing, SEC v. Kik case AND the unbalance of huge supply v. low demand. The development of KIN Ads and the usage of Code Wallet when it happens will balance the situation regarding demand.
All other Alts have been down for some time. These are factual reasons why KIN is at current pricing level

**3.	By stoping giving them KIN they have a ngeative delta Supply vs Demand, meaning every month their supply of KIN will decrease until eventually they either need to refill their supply by buying it of exchanges or by decreasing the amount of kin giving to users for their earn and spends. Alone by these two factors you increase the value of kin externally on exchanges and internally within the apps. On top changing to the 3 variables I mentioned such as fiat spend for kin, you truley rewards those apps who bring real demand to the table.**
OK so here, you are clearly stating that the partners should not receive any KIN at all. A reduction of KIN supply on the market (by stopping the distribution of KIN via the KRE) is a decrease of KIN supply overtime.
You talk about facts and numbers but here, you assume that the big partners will buy KIN when their Kin reserves will shrink. It is a bet to me. Don’t you think that the risk that the partners leaving the Ecosystem after they do not have KIN/dollars anymore is too big? The value of KIN will increase if the partners get no KIN and distribute less KIN and if they will buy from exchanges when their Kin reserve will diminish but you do not know about it. I can see a specific grant in USD to be given to the Apps if they commit to develop real demand functions in their Apps but even this is not a long term sustanaible approach. 

**4.	You do realize all the countries who used dollarization are examples of failed economies, I underline for you failed!!! The only way they were able to continue was to introduce dollars. I cannot ensure that KIN will thrive after this as the duration period or the amount might be to small, but what I definetly can say my approach will focus truley on the demand side and will bring back value to kin. I repeat once we have an upgoing circle it has positve external effects acting as a catalyst for the ecosystem, bring back speculators, bringing new apps and users**
Yes you are 100% correct. So to me, choosing to reward the Ecosystem Apps in dollars would mean that the Ecosystem failed and to me, it is far to be the case. Choosing your approach will make the project do a “Hail Mary” and will be considered by many as a way to find a way to pump the price.
I do not have a crystal ball but I would like to see the situation after the trial, after the Solana Migration and after the Code wallet usage regarding demand.
Again, I am not against finding a solution to increase the demand but your proposal, to me, does not seem to be the correct approach. This would bring some “one-shot”, “artificial” short-term measure and then, once the USD will not be distributed, what will happen? 
You are making the bet that the artificial value provided by the USD distributed will attract more people to the project and because Kin will have a bigger price, it will attract other people to join. Your approach is focused on attracting speculators, you said it yourself. So meaning, counting on speculative demand which is less reliable then real demand.

**5.	You basically spoted that kin is not a true medium of exchange , which you perfectly explained not even in its very own ecosystem, this however would immediately change if had a constant growing value , than beyond the ecosystem it could be used outside in the real world for example for a coffee or a gift card etc... Would you not agree if you had an app and you know every month the kin I hold is worth more, would you not create as much spends as possible in order to get the max of kin inflow as possible, knowing that the kin will be much more worth the next month. If you have buisness sense and want a return then you would do it**
Yes obviously, if there would be more partners, more Apps and more varied incentives to spend your KIN, there would be more tourism/transfers. For now, why would I bother to move my KIN from one App to another? I am sure that the more partners, the more KIN transfers will happen. So higher KIN value will make KIN moved more from one App to another? Possibly. It might be that a higher KIN value will increase more the movements of KIN from wallets to Exchanges. 
Regarding your remark about spends, I am also not sure. Despite the value of KIN during the last past two years, spends haven’t been that developed enough by the developers. Why so? I don’t have the answer and it is a pity. From my perspective, I believe that KIN distributed to the Partners through the KIN KRE should have been done also with the condition of Apps proposing valid amount of spends. 
Low number of spends -> lower KIN distributed through the KRE but this is more about fixing the supply and not the demand. 
My business sense tells me that it is not black or white. A lot of variables are at hand here and giving dollars as a fix proposal in order to rebalance the disequilibrium between supply and demand will not have a real effect on the long run.

**6.	Wrong please read again, I never asked in a single line for a supply reduction or a coin burn, I asked for stop KIN payouts wotsoever and convert them into dollars. Stop spreading false information**
That’s actually what you did. You explicitly stated in 3/ the following 
“By stoping giving them KIN they have a ngeative delta Supply vs Demand, meaning every month their supply of KIN will decrease “

**7. Its an equation and eventually they have to reinvest their dollars and buy KIN as they run dry. The point is not only to switch the KRE to dollars but only rewards those who push demand with fiat spend for kin. The so created positiv delta will eventually catch up with the supply and will force the apps to reinvest, which then starts the upward trend in terms of a sustainable increase of kin value and demand. You are right they could aquire as much dollars as possible and exit, but to be fair they can do similar wtih kin aquire as much kin as possible and exit. And to be fair some apps have done so by not even have switched to the kin blockchain. Would you not agree the incentive for an app is greater to stay within the ecosystem and be pro active rewarding them somehing of value instead of an asset which has only 0,13% of its ATH value left. You have to think also a step ahead that as more they focus on the spends and more the denand increases the more the value of kin rises meaning ultimately the econimical benefit using kin will increase for each app, if you want so the econimical benefit of each app using kin right now is only 0.13% what is was once before.**
No they do not have to. They might leave the Ecosystem as well and we had a case not long ago. Remember Vent leaving all of a sudden due to a Business decision? You assume that partners will reinvest the USD in KIN but you do not have the confirmation they will do it. Here you are not factual and are building your logic based on an assumption.
Some Apps have not switched to KIN3, it is a fact. There should have been a limitation of KIN distributed to those partners by the KRE imo (reducing supply) as they were not 100% migrated to the KIN blockchain (which would have made a tiny increase of demand)
Regarding rewarding Apps that are fostering and promoting spends to make real demand increase, I am 100% in line with you and I agree. But again giving them dollars in parallel can be not sustanaible on the long run. You would reward those Apps in dollars for a limited period of time? 
Besides, from a compliance standpoint and based on the transparency document, the KIN budget is defined in KIN only. Rewarding dollars to partners via the KRE is not mentioned. 

8.**Yes I certainly believe it will help to create more adoption and kin awareness. It is not crazy to believe that distributing FIAT could bring new apps to the ecosystem. Especially new apps are the one who need to rebuy KIN which would you instanly create a function economical circle. I cannot speak about other projects who might be more suitable for whatever reason. May be others reward more, but KIN does have a funtiong network with cross joined apps , which is appealing to medium seized apps.**
Distributing FIAT might attract people and developers. But what will attract much more partners is the image, reputation, potential, adoption, vision of the project. Price would attract speculators. If KIN has no speculative demand now is mostly because of the lack of exchanges and the SEC v. Kik case. Also a lack of marketing due to the fact that we need all the SEC Cloud dissipated and all technical aspects tackled first.
Fundamentals. If fundamentals are good, it will attract naturally people and partners. The SEC v Kik case finished, a lot of things will move forward. Solana mainnet golive will probably be done at the same time most of the KIN Ecosystem will be migrated on the Solana Network.

**9.	Wrong it is not a reduction it is setting the supply fixed for now until we meet equilibrium**
You are playing with words here. 
Stopping the KIN supply by stopping the KIN distributed through the KRE is a reduction of supply overtime. If you stop distributing KIN via the KRE during the next period, about 3b KIN will not be distributed. So it IS a reduction of KIN supply.

**10.	This is your personal view based on what facts? I provided you a concept both proven by history and econimcs. Demand will never ever happen in the current state, as again the incentive to buy , hold or earn kin is so little, as a direct result of the hyperinflation. the only way of stoping hyperinflation is puting pressure on the supply and increase the value of kin which then will bring the above asked demand combined with your oganical growth the ecosystem will eventually come to an equlibrium where rewards as such wont be neccessary.**
Not quite. Demand is happening. Kin Ads will be deployed more and more. That’s concrete AND sustainable for the Ecosystem. The proposal for paying Exchanges, if adopted, would also be a contribution towards demand. Demand is happening. Again, not black or white. Code Wallet to-be deployed 3Q2020…
Reducing hyperinflation could be done with the following: Adjusting the KIN KRE distribution and put as a parameter in the formula the development of spends in App as a condition -> limit the distribution of KIN if the top Apps do not adapt (reducing suppl and the positive consequence,. if Apps still want to receive more KIN, they would need to develop more spends.

**11.	As said in the proposal per month we reward 200.000 dollars distributed to apps accordingly to their share of the 3 presented variables where fiat spend for kin is the most important. It is a subzidized ecosystem focusing only on the demand leading to an increase of value of kin. It is not artificial as the increase of value comes from within the ecosystem from its very own demand. Combined with a change of incentive to hold kin , user, speculator and devs will increase their demand to acquire a portion x of KIN. It is exactly the oposite what the KRE does now , instead of subzidizing liquidity we subsidize true demand.**
You can write it in multiple different ways, it does not matter. This would work only if the Ecosystem Apps will willingly and freely buyback KIN with the received dollars. If they HAVE TO buy KIN contractually with the received FIAT (or even part of it) then it would be different and I could consider the idea further. But then again, what happens if KF is running out of money?
I was advocating for staking system/interest rate based on the KIN stash that any user would hold for a specific amount of time on a specific address but this could have been interpreted negatively by the SEC and hence, was not discussed further. 
Who knows, one day, if KIN is officially recognized as a currency, then we might see people buying more KIN in order to hold more KIN and in order to receive an interest based on their KIN stash.

**I repeat it is not about the price of kin, it is about abolishing hyperinflation and rewarding demand and increasing demand and yes this means kins price should not be 0,13% what it used to be.
I bet all my kin, if KIN would be close to ATH price levels, the ecosystem and the adoption of kin would be at least 10x bigger. Each app, user, speculator would have had a 93x higher incentive in terms of value to hold buy or use kin, just think about 93x times. What do you or anyone here do not understand , it is plane simple right there.**
Higher price as an incentive to attract more investors is obvious. I can imagine an investor willingly investing in a top100 coin instead of investing in a top 500 coin to mitigate the risks. But price will be the positive consequence on growing organic demand from the Apps. 
I understand what you mean. Your proposal to boost demand has the merit to exist and might be considered by KF.

**Comment by Throwawayburros** [comment]
> We know that KF will get money from migrating to Solana and we know that KF had a buy back program over 1 Million dollars , how much was spend I cannot say.

The proposal says the SOL that KF earns, is on a 1 year delay.  If KiK was as rich as you think, they wouldn't be selling KiK to fund the SEC case.  In the past KiK was $1 billion dollar company, and I can assure you that if MediaLabs was the only company interested, then MediaLabs can pay as little as they want.  Rumor websites were reporting the deal was for less than $10 million.

> Show me the source and the evidence that it is so?

Ted mentioned that the SEC has warned exchanges not to list Kin, and thats why we have not had any new exchange listings for 2 years.  Before the SEC case, Kin was full of speculators causing the price to go up and down, such as will Kin be on Coinbase.  Will Kin be added to Snapchat.  Will Kin be added to Binance.  Will Kin be added to Zynga.  We have not had any partners as big as the community was expecting Without the speculation aspect, its just going to be a long slow steady decline.  

> I can certainly say since the KRE started the trading pair of kin droped from an average of 0.0000030 ETH to 0.00000003 ETH, when the SEC announcement came KIN was actually at levels 0.00000040 eth and higher and dropped to arround 0.0000030 ETH which is a sharp drop

Your memory is playing tricks on you.
Kins trading price was around 0.00000014 ETH ($0.00003555) before the SEC news came out on June 3rd 2019.  Then it dropped to 0.00000010 ETH.  [Verify it here.](https://www.coingecko.com/en/coins/kin)   
The SEC press release occurred on June 4th.  https://www.sec.gov/news/press-release/2019-87

June 3rd 2019 Kin = $0.00003555
June 4th 2019 SEC news becomes public
June 5th 2019 Kin = $0.00002585

> First of all this is KIN and not Ripple , two different blockchains two different approach, two different approaches. 

They are very similar when you expand the token supply to be equal to Kin.  You believe that the KRE is dumping too many coins on the market, which is the root cause of the low price.  But I would also argue that [Ripple has the same issue in Jed McCaleb](https://cointelegraph.com/news/ripple-co-founder-jed-mccaleb-sold-a-billion-xrp-and-has-47b-left).  Jed has been dumping around 2% of the total supply every year, for years in a similar fashion to how the KRE devs are dumping large amounts of the KRE.  The difference, like I mentioned, is that the ripple subreddit is full of speculators, talking about which banks are going to adopt XRP, speculation about XRP disrupting swift, speculating about XRP disrupting the remittance market.  We have zero of that for Kin.  All the speculation is gone.  The speculation now is "Will the SEC use lube or no lube for when they screw us?"

**Comment by Raketenernie** [comment]
> **1. This is partly true but you just quoted the syntomes as a result of the hyperinflation itself, descriping what is lacking does not focus on the core problem nor solves it the core problem**
> Don’t get me wrong. Anything that can increase demand is good for the project. I just doubt your approach is the correct way of doing it. KIN usage and spends from more users and users buying KIN massively will make the real demand increase gradually. This is the core problem. I don’t think that a short-term KRE paid in dollars would fix the systemic issue that we currently have (lack of real demand) but your contribution is at least an input to discuss demand

 As said I do not have a crystall ball , how it fast will work, but it will eventually work thats what subsisides do stimulating the market.

> **2. Again you so hooked up on the syntome of a low value of kin, we have to tacle the problem why kin has a low value and yes mainly it is because of the free inflow of KIN. This is matter of fact, the statisitics show this fact**
> As you are also following KIN for a long time, the current low price of KIN is the consequence of multiple factors. Pantera Capital dumping Billions of KIN, crypto market crashing, SEC v. Kik case AND the unbalance of huge supply v. low demand. The development of KIN Ads and the usage of Code Wallet when it happens will balance the situation regarding demand.
> All other Alts have been down for some time. These are factual reasons why KIN is at current pricing level,. 
I agrree here with you that kins price / value is the combination of many variables, but recent price drop by factor 10 and more is due to KRE.  Yes Kin adds will bring additional dollars to the ecosystem, but since I have not even seen a tiny info about projected returns , a ratio of return of ads per user it is totally unclear how much ads will bring is it 1.000 dollars per months or 100.000 dollars. Knowing the apps who are using KIN, knowing the very infant stage of kin ,I assume it wont be a lot of new free dollars. Yet again we are at the same problem you even mentioned, who says that apps making money with ads will spend it for kin, as if we stay at status quo they will receive more kin over the KRE than they need. 
> 
> **3. By stoping giving them KIN they have a ngeative delta Supply vs Demand, meaning every month their supply of KIN will decrease until eventually they either need to refill their supply by buying it of exchanges or by decreasing the amount of kin giving to users for their earn and spends. Alone by these two factors you increase the value of kin externally on exchanges and internally within the apps. On top changing to the 3 variables I mentioned such as fiat spend for kin, you truley rewards those apps who bring real demand to the table.**
> OK so here, you are clearly stating that the partners should not receive any KIN at all. A reduction of KIN supply on the market (by stopping the distribution of KIN via the KRE) is a decrease of KIN supply overtime.
> You talk about facts and numbers but here, you assume that the big partners will buy KIN when their Kin reserves will shrink. It is a bet to me. Don’t you think that the risk that the partners leaving the Ecosystem after they do not have KIN/dollars anymore is too big? The value of KIN will increase if the partners get no KIN and distribute less KIN and if they will buy from exchanges when their Kin reserve will diminish but you do not know about it. I can see a specific grant in USD to be given to the Apps if they commit to develop real demand functions in their Apps but even this is not a long term sustanaible approach.
> Well sooner or later they have to buy kin as they run out. If not their app with earns and spends would stop working. So their choice would be , stay and buy kin or leave kin. Leaving kin would meanw they would lose the mighty great joined network  of over 4 mio activer users + losing the dollar rewards. The threat any app would drop out is now greater than in my scenario / apporach. Now every app has the mighty network but receives tokens of worthless value, whereas if they would get dollars the incentive would be greater. Here you speak from a very KF  view approach, you so feared that apps could drop out once they run out and to be honest this is not unrealistic at all. But I ask you the apps who use kin who are only in for free cash / rewards ( subsidies and do a runner as soon they have to give something (moentary ) back are these the apps who help kin grow? Are these the apps pushing adoption? Are these the apps pushing the idea of a joined ecosystem? Are these the apps we should sponsor? Should we not sponsor the medium / small sized apps who live and push the idea of kin as kimeo for example? Yes the big ones they have the numbers they got reputation what does it help kin to have them if they are only in for the "free" money grab? That is why if we would reward fiat spend for kin, it would be time for all apps and devs to show true colour if they believe in kin and push or they just in for the money. Even if  it would mean we lose 3 million MAE but the remaining 1 MAE are the ones being heavly pushed to use kin, buy , earn and spend it by apps who believe in kin , the idea the tech would you not think that this ecosystem would be sustainable and has a real value? 

> **4. You do realize all the countries who used dollarization are examples of failed economies, I underline for you failed!!! The only way they were able to continue was to introduce dollars. I cannot ensure that KIN will thrive after this as the duration period or the amount might be to small, but what I definetly can say my approach will focus truley on the demand side and will bring back value to kin. I repeat once we have an upgoing circle it has positve external effects acting as a catalyst for the ecosystem, bring back speculators, bringing new apps and users**
> Yes you are 100% correct. So to me, choosing to reward the Ecosystem Apps in dollars would mean that the Ecosystem failed and to me, it is far to be the case. Choosing your approach will make the project do a “Hail Mary” and will be considered by many as a way to find a way to pump the price.
> I do not have a crystal ball but I would like to see the situation after the trial, after the Solana Migration and after the Code wallet usage regarding demand.
> Again, I am not against finding a solution to increase the demand but your proposal, to me, does not seem to be the correct approach. This would bring some “one-shot”, “artificial” short-term measure and then, once the USD will not be distributed, what will happen?
> You are making the bet that the artificial value provided by the USD distributed will attract more people to the project and because Kin will have a bigger price, it will attract other people to join. Your approach is focused on attracting speculators, you said it yourself. So meaning, counting on speculative demand which is less reliable then real demand.

Well I got to say you got some valid points which made me already think to improve my proposal. The word artifical pump is not correct, as it is a subsidy similar as the KRE by giving KIN. But here we give dollars for demand. The price is not a direct pump as we do not go to exchanges and just pump it. We will reward on a regular basis the ones (apps) who pushed demand. This very demand isat the moment 200 Million kin spend per day. This is the demand from the ecosystem which will push the price. Therefore this demand has nothing to do with speculation. Additionally and here you are right it will bring back speculators and lvolume back to exchanges. Everything which kin is lacking at the moment. This will make it again for apps appealing to stay with kin as they can faster sell or buy kin, where as now only buy is truly possible. Last but not least users will come bk and more new users will join cuz there will be a growing an incentive to buy and hold kin.
The SEC outcome does play a very crucial role on the future months of kin, I am very pessimistic if kin loses and very optimistic if kin wins. If kin wins and you switch the kre to my demand approach combined with the migration, code and the ads there will be alot of variables pushing demand, we could create a hype will we limiting / stoping and increase of supply. If we continue with the KRE the negative delta of the payouts and the already exiting negative NDB could destroy any upward trend, Just a reminder 300 billiion kin have to be asorbed just to be back at equilibrium of supply and demand-where as 200 million per day is covered up by the ecosystem the rest has to come from speculators, so you see we have to do something vs this unbalance.
> 
> **5. You basically spoted that kin is not a true medium of exchange , which you perfectly explained not even in its very own ecosystem, this however would immediately change if had a constant growing value , than beyond the ecosystem it could be used outside in the real world for example for a coffee or a gift card etc... Would you not agree if you had an app and you know every month the kin I hold is worth more, would you not create as much spends as possible in order to get the max of kin inflow as possible, knowing that the kin will be much more worth the next month. If you have buisness sense and want a return then you would do it**
> Yes obviously, if there would be more partners, more Apps and more varied incentives to spend your KIN, there would be more tourism/transfers. For now, why would I bother to move my KIN from one App to another? I am sure that the more partners, the more KIN transfers will happen. So higher KIN value will make KIN moved more from one App to another? Possibly. It might be that a higher KIN value will increase more the movements of KIN from wallets to Exchanges.
> Regarding your remark about spends, I am also not sure. Despite the value of KIN during the last past two years, spends haven’t been that developed enough by the developers. Why so? I don’t have the answer and it is a pity. From my perspective, I believe that KIN distributed to the Partners through the KIN KRE should have been done also with the condition of Apps proposing valid amount of spends.
> Low number of spends -> lower KIN distributed through the KRE but this is more about fixing the supply and not the demand.
> My business sense tells me that it is not black or white. A lot of variables are at hand here and giving dollars as a fix proposal in order to rebalance the disequilibrium between supply and demand will not have a real effect on the long run. You said it yourself the spends where not pushed or not enough, not appealing enough not sustainable enough, why was that? It was the KRE rewarded any activity including earns. So there was no need to focus on spends, especially for the apps (we come to what I wrote earlier) who are only in for the free money and not the tech and idea of kin. Cmon we have top tier app still on the ETH blockchain what does it tell us about their true motives using kin, you can tell me what you want they definetly 100 passionate and full in with kin. To come back by rewarding dollars for fiat spend on kin, the apps have to create spends and ways for users to buy kin / top up (over direct fiat gateway --> let is be code so we could keep track for the rewards) in order to a) receive a nice junk of the reward and b) by doing so they themselves create real demand pushing the value of kin , meaning that their turnover & profit increases by the same factor kins value has increased. Since it is a joined ecosystem means that all benefit form a value increase of kin 
> 
> **6. Wrong please read again, I never asked in a single line for a supply reduction or a coin burn, I asked for stop KIN payouts wotsoever and convert them into dollars. Stop spreading false information**
> That’s actually what you did. You explicitly stated in 3/ the following
> “By stoping giving them KIN they have a ngeative delta Supply vs Demand, meaning every month their supply of KIN will decrease “ I am not reducing the supply , I am stopping further increase in the supply, meaning the avialiable supply of kin in the market will decrease, the overall circulating supply will continue to be 1.5 trillion, but since more ppl will buy and hold kin it will be more distributed over more user, speculators and apps. This is what we should want. Why do I want to do this as quoted before the supply and demand of kin is not in equilibrium we have a positive delta on the supply of 300 billion kin, which needs to be absorbed by the demand , which is as low of 200 million kin spend per day (in the ecosystem). So it means we have to reduce the oversupply of the 300 billion kin and the least you want to do put even more kin supply on top to make it even bigger. That is why stop supply inflow so you can subtract at least 200 million kin per day of this oversupply. That alone would take 4 years that is why you need to subsidize demands to generate extra demand to absorb faster the 300 billion kin. This is all under the assumption ceteris paribus, if the ecosystem would grow the oversupply could shrink faster or vice versa it could take longer. But we have to tacle the oversupply, it is a hugh oversupply which can kill any kin of upward trend, kill will sit for years on currents value if not lower
> 
> **7. Its an equation and eventually they have to reinvest their dollars and buy KIN as they run dry. The point is not only to switch the KRE to dollars but only rewards those who push demand with fiat spend for kin. The so created positiv delta will eventually catch up with the supply and will force the apps to reinvest, which then starts the upward trend in terms of a sustainable increase of kin value and demand. You are right they could aquire as much dollars as possible and exit, but to be fair they can do similar wtih kin aquire as much kin as possible and exit. And to be fair some apps have done so by not even have switched to the kin blockchain. Would you not agree the incentive for an app is greater to stay within the ecosystem and be pro active rewarding them somehing of value instead of an asset which has only 0,13% of its ATH value left. You have to think also a step ahead that as more they focus on the spends and more the denand increases the more the value of kin rises meaning ultimately the econimical benefit using kin will increase for each app, if you want so the econimical benefit of each app using kin right now is only 0.13% what is was once before.**
> No they do not have to. They might leave the Ecosystem as well and we had a case not long ago. Remember Vent leaving all of a sudden due to a Business decision? You assume that partners will reinvest the USD in KIN but you do not have the confirmation they will do it. Here you are not factual and are building your logic based on an assumption.
> Some Apps have not switched to KIN3, it is a fact. There should have been a limitation of KIN distributed to those partners by the KRE imo (reducing supply) as they were not 100% migrated to the KIN blockchain (which would have made a tiny increase of demand)
> Regarding rewarding Apps that are fostering and promoting spends to make real demand increase, I am 100% in line with you and I agree. But again giving them dollars in parallel can be not sustanaible on the long run. You would reward those Apps in dollars for a limited period of time?
> Besides, from a compliance standpoint and based on the transparency document, the KIN budget is defined in KIN only. Rewarding dollars to partners via the KRE is not mentioned.
I answered before here my view, these are not true partners and we should not support them, one thing which could engage apps to stay and especially hold kin is add to my approach staking but also in dollars. I thought about it there is so much liquduity it will take a very long time that demand can overcome the 300 billion kin , unless 300 of us buy 1 billion kin each may be an idea. Staking could help that apps not dump all and cash out and do a runner, have a second incentive to stay, hold and aquire kin.
> 
> 8.**Yes I certainly believe it will help to create more adoption and kin awareness. It is not crazy to believe that distributing FIAT could bring new apps to the ecosystem. Especially new apps are the one who need to rebuy KIN which would you instanly create a function economical circle. I cannot speak about other projects who might be more suitable for whatever reason. May be others reward more, but KIN does have a funtiong network with cross joined apps , which is appealing to medium seized apps.**
> Distributing FIAT might attract people and developers. But what will attract much more partners is the image, reputation, potential, adoption, vision of the project. Price would attract speculators. If KIN has no speculative demand now is mostly because of the lack of exchanges and the SEC v. Kik case. Also a lack of marketing due to the fact that we need all the SEC Cloud dissipated and all technical aspects tackled first.
> Fundamentals. If fundamentals are good, it will attract naturally people and partners. The SEC v Kik case finished, a lot of things will move forward. Solana mainnet golive will probably be done at the same time most of the KIN Ecosystem will be migrated on the Solana Network.
> 
You are right but to be fair kins reputation is that of a shit coin, as much I love this project its reputation is burned. Speculator were send to hell and they are gone but the truth is you need their capital to push a project, kin approach of focusing only on users was a false strategy it came a decade to early. In the infant stage of an advancing technology you need a lot of capital, best example is Tesla, look how much capital they collected and not even had a finised product to be produced on mass. I blieve if SEC turns out good, solona can do what is promised code looks fine there will positive momentum, but you still have strong opponents mr hyperinflation and the oversupply.Especially the oversupply will swallow up alot of the positive momentum. 
> **9. Wrong it is not a reduction it is setting the supply fixed for now until we meet equilibrium**
> You are playing with words here.
> Stopping the KIN supply by stopping the KIN distributed through the KRE is a reduction of supply overtime. If you stop distributing KIN via the KRE during the next period, about 3b KIN will not be distributed. So it IS a reduction of KIN supply. Its a delay , eventually once we are back in equilibrium kin can be granted again, accordingly the resilence of the ecosystem.
> 
> **10. This is your personal view based on what facts? I provided you a concept both proven by history and econimcs. Demand will never ever happen in the current state, as again the incentive to buy , hold or earn kin is so little, as a direct result of the hyperinflation. the only way of stoping hyperinflation is puting pressure on the supply and increase the value of kin which then will bring the above asked demand combined with your oganical growth the ecosystem will eventually come to an equlibrium where rewards as such wont be neccessary.**
> Not quite. Demand is happening. Kin Ads will be deployed more and more. That’s concrete AND sustainable for the Ecosystem. The proposal for paying Exchanges, if adopted, would also be a contribution towards demand. Demand is happening. Again, not black or white. Code Wallet to-be deployed 3Q2020…
> Reducing hyperinflation could be done with the following: Adjusting the KIN KRE distribution and put as a parameter in the formula the development of spends in App as a condition -> limit the distribution of KIN if the top Apps do not adapt (reducing suppl and the positive consequence,. if Apps still want to receive more KIN, they would need to develop more spends. Look I bet you the top tier 5 apps having billions of kin, do need not more kin, they would be happy to receive dollars. To receive dollars they have to push spends sounds to me good. Kin Ads, we do not know how significant that will be, I fear it will be as mighty as several thousands dollars of fiat spend for kin per month, a metric for example KF hardly ever published as it is so significant small that is shows the illness of the ecosystem. PAying exchanges to list kin , to get on binance and bitfinex and coinbase hm may be I can agree on that but that brings nothing sustainable to the table , rather I would like to see staking or that apps list kin cuz they like kin not because we paid us, cuz these are exchanges who will drop kin as soon there is nothing in for them. 
> 
> **11. As said in the proposal per month we reward 200.000 dollars distributed to apps accordingly to their share of the 3 presented variables where fiat spend for kin is the most important. It is a subzidized ecosystem focusing only on the demand leading to an increase of value of kin. It is not artificial as the increase of value comes from within the ecosystem from its very own demand. Combined with a change of incentive to hold kin , user, speculator and devs will increase their demand to acquire a portion x of KIN. It is exactly the oposite what the KRE does now , instead of subzidizing liquidity we subsidize true demand.**
> You can write it in multiple different ways, it does not matter. This would work only if the Ecosystem Apps will willingly and freely buyback KIN with the received dollars. If they HAVE TO buy KIN contractually with the received FIAT (or even part of it) then it would be different and I could consider the idea further. But then again, what happens if KF is running out of money?
> I was advocating for staking system/interest rate based on the KIN stash that any user would hold for a specific amount of time on a specific address but this could have been interpreted negatively by the SEC and hence, was not discussed further.
> Who knows, one day, if KIN is officially recognized as a currency, then we might see people buying more KIN in order to hold more KIN and in order to receive an interest based on their KIN stash. As said staking is not a bad idea but combined with the adjustment on the KRE. Getting interest on kin does not make it a security, money give you interest /return too. I just had an idea while you were saying why would apps willingly and freely buyback kin, cuz a) they run dry and they have to buy or b) we give them additional incentive in form of a reward (meaning they gain a abitrage advantage)--> extra profit for them
> 
> **I repeat it is not about the price of kin, it is about abolishing hyperinflation and rewarding demand and increasing demand and yes this means kins price should not be 0,13% what it used to be. I bet all my kin, if KIN would be close to ATH price levels, the ecosystem and the adoption of kin would be at least 10x bigger. Each app, user, speculator would have had a 93x higher incentive in terms of value to hold buy or use kin, just think about 93x times. What do you or anyone here do not understand , it is plane simple right there.**
> Higher price as an incentive to attract more investors is obvious. I can imagine an investor willingly investing in a top100 coin instead of investing in a top 500 coin to mitigate the risks. But price will be the positive consequence on growing organic demand from the Apps.
> I understand what you mean. Your proposal to boost demand has the merit to exist and might be considered by KF.

I am not saying my proposal is pefect or it will work they way I beleive, but it should be clear for everyone, we have oversupply and hyperinflation and this does not encourage, to use, hold or buy kin. We have to create incentives fixing this core problem.

1. Look at the variables we reward
2. Look what currency we reward and the amount
3. Look who we reward and what is their true value for the ecosystem
4. Look for ways to bring demand and supply back to equlibrium
5. Look who and how we can bring econimcal measurable value to kin 

I will adjust my proposal when I have time I will add a FAQ section. By the end of the day if we start reward true demand that would be a start even tho I strongly beleive subsidizing demand will do a lot, a lot more than paying exchanges , doing a direct buy back etc... as we would subsidize the ecosystem.

**Comment by Polype01** [comment]
1.	Raketenernie: This is partly true but you just quoted the syntomes as a result of the hyperinflation itself, descriping what is lacking does not focus on the core problem nor solves it the core problem
Polype01: Don’t get me wrong. Anything that can increase demand is good for the project. I just doubt your approach is the correct way of doing it. KIN usage and spends from more users and users buying KIN massively will make the real demand increase gradually. This is the core problem. I don’t think that a short-term KRE paid in dollars would fix the systemic issue that we currently have (lack of real demand) but your contribution is at least an input to discuss demand

Raketenernie: As said I do not have a crystall ball , how it fast will work, but it will eventually work thats what subsisides do stimulating the market.

Polype01: OK. But here, you said before that it is a fact and that it will work. Anyway, I agree with you if applied, we do not know if this would work and when this would work

2.	Raketenernie: Again you so hooked up on the syntome of a low value of kin, we have to tacle the problem why kin has a low value and yes mainly it is because of the free inflow of KIN. This is matter of fact, the statisitics show this fact
Polype01: As you are also following KIN for a long time, the current low price of KIN is the consequence of multiple factors. Pantera Capital dumping Billions of KIN, crypto market crashing, SEC v. Kik case AND the unbalance of huge supply v. low demand. The development of KIN Ads and the usage of Code Wallet when it happens will balance the situation regarding demand.
All other Alts have been down for some time. These are factual reasons why KIN is at current pricing level,. 
Raketenernie: I agrree here with you that kins price / value is the combination of many variables, but recent price drop by factor 10 and more is due to KRE. Yes Kin adds will bring additional dollars to the ecosystem, but since I have not even seen a tiny info about projected returns , a ratio of return of ads per user it is totally unclear how much ads will bring is it 1.000 dollars per months or 100.000 dollars. Knowing the apps who are using KIN, knowing the very infant stage of kin ,I assume it wont be a lot of new free dollars. Yet again we are at the same problem you even mentioned, who says that apps making money with ads will spend it for kin, as if we stay at status quo they will receive more kin over the KRE than they need.
Polype01: To be fair, I would estimate that KIN dumping from the Partners has been causing about 30% of the price erosion. The rest are external factors (Pantera, market crashes, SEC..)
I have no specific, concrete numbers for Kin Ads but the impact of it can have a reasonable, positive impact on Demand. It will certainly be more than 1000USD from what I understand and not 100000 USD (one can dream lol)
 Josh to give an estimate of revenues that would impact demand might follow up soon
3.	Raketenernie:  By stoping giving them KIN they have a ngeative delta Supply vs Demand, meaning every month their supply of KIN will decrease until eventually they either need to refill their supply by buying it of exchanges or by decreasing the amount of kin giving to users for their earn and spends. Alone by these two factors you increase the value of kin externally on exchanges and internally within the apps. On top changing to the 3 variables I mentioned such as fiat spend for kin, you truley rewards those apps who bring real demand to the table.
Polype01: OK so here, you are clearly stating that the partners should not receive any KIN at all. A reduction of KIN supply on the market (by stopping the distribution of KIN via the KRE) is a decrease of KIN supply overtime.
You talk about facts and numbers but here, you assume that the big partners will buy KIN when their Kin reserves will shrink. It is a bet to me. Don’t you think that the risk that the partners leaving the Ecosystem after they do not have KIN/dollars anymore is too big? The value of KIN will increase if the partners get no KIN and distribute less KIN and if they will buy from exchanges when their Kin reserve will diminish but you do not know about it. I can see a specific grant in USD to be given to the Apps if they commit to develop real demand functions in their Apps but even this is not a long term sustanaible approach.
Raketenernie: Well sooner or later they have to buy kin as they run out. If not their app with earns and spends would stop working. So their choice would be, stay and buy kin or leave kin. 
Polype01: You are correct.
Raketenernie:  Leaving kin would meanw they would lose the mighty great joined network of over 4 mio activer users + losing the dollar rewards. The threat any app would drop out is now greater than in my scenario / apporach. Now every app has the mighty network but receives tokens of worthless value, whereas if they would get dollars the incentive would be greater. 
Polype01: I do not agree here. Even if KIN has a low value now, it has a value. The sum of all KIN KRE distributed is worth 1,000,000 USD combined since the KIN KRE inception at today price. If KIN valuation is 10x higher, the KIN KRE value would be 10,000,000 USD. 
Would we have the same discussion whether KIN price would be 10x higher? I would say yes as the current systemic unbalance (Enormous supply vs very low demand) would still be there.
Raketenernie: Here you speak from a very KF view approach, you so feared that apps could drop out once they run out and to be honest this is not unrealistic at all. But I ask you the apps who use kin who are only in for free cash / rewards ( subsidies and do a runner as soon they have to give something (moentary ) back are these the apps who help kin grow? 
Polype01: Man, I Fully agree with you. An Ecosystem App should not be there just to receive KIN from KRE and not contributing to the KIN Ecosystem development. I hate to see big partners getting 90% of the KIN KRE every week and see not strategic decision from them to develop spends and make KIN really used and pushed. 
Raketenernie: Are these the apps pushing adoption? Are these the apps pushing the idea of a joined ecosystem? Are these the apps we should sponsor? Should we not sponsor the medium / small sized apps who live and push the idea of kin as kimeo for example? 
Polype01: Yes!! Definitely
Raketenernie: Yes the big ones they have the numbers they got reputation what does it help kin to have them if they are only in for the "free" money grab? That is why if we would reward fiat spend for kin, it would be time for all apps and devs to show true colour if they believe in kin and push or they just in for the money. Even if it would mean we lose 3 million MAE but the remaining 1 MAE are the ones being heavly pushed to use kin, buy , earn and spend it by apps who believe in kin , the idea the tech would you not think that this ecosystem would be sustainable and has a real value?
Polype01: Your view is to reward dollars to the Apps who will be pushing the KIN vision and adoption within the Ecosystem. A periodic, limited in time grant in USD.
My view is to limit the amount of KIN distributed to them if they do not contribute more to the development of the KIN philosophy within the Ecosystem and rebalance the KIN KRE by redistributing more KIN to the smaller Apps and to the Apps that are really trying hard to implement the KIN philosophy. I do believe the Kin Ads and Code will smoothly solve the demand issue but also, to me the problem is that no newer partner joined the Ecosystem lately and the number of active Apps is around 45%. We need more partners and we do not have this happening due to the SEC case

4. Raketenernie: You do realize all the countries who used dollarization are examples of failed economies, I underline for you failed!!! The only way they were able to continue was to introduce dollars. I cannot ensure that KIN will thrive after this as the duration period or the amount might be to small, but what I definetly can say my approach will focus truley on the demand side and will bring back value to kin. I repeat once we have an upgoing circle it has positve external effects acting as a catalyst for the ecosystem, bring back speculators, bringing new apps and users
Polype01: Yes you are 100% correct. So to me, choosing to reward the Ecosystem Apps in dollars would mean that the Ecosystem failed and to me, it is far to be the case. Choosing your approach will make the project do a “Hail Mary” and will be considered by many as a way to find a way to pump the price.
I do not have a crystal ball but I would like to see the situation after the trial, after the Solana Migration and after the Code wallet usage regarding demand.
Again, I am not against finding a solution to increase the demand but your proposal, to me, does not seem to be the correct approach. This would bring some “one-shot”, “artificial” short-term measure and then, once the USD will not be distributed, what will happen?
You are making the bet that the artificial value provided by the USD distributed will attract more people to the project and because Kin will have a bigger price, it will attract other people to join. Your approach is focused on attracting speculators, you said it yourself. So meaning, counting on speculative demand which is less reliable then real demand.
Raketenernie: Well I got to say you got some valid points which made me already think to improve my proposal. The word artifical pump is not correct, as it is a subsidy similar as the KRE by giving KIN. But here we give dollars for demand. The price is not a direct pump as we do not go to exchanges and just pump it. We will reward on a regular basis the ones (apps) who pushed demand. This very demand isat the moment 200 Million kin spend per day. This is the demand from the ecosystem which will push the price. Therefore this demand has nothing to do with speculation. Additionally and here you are right it will bring back speculators and lvolume back to exchanges. Everything which kin is lacking at the moment. This will make it again for apps appealing to stay with kin as they can faster sell or buy kin, where as now only buy is truly possible. Last but not least users will come bk and more new users will join cuz there will be a growing an incentive to buy and hold kin.
Polype01: I agree again with what could happen but it is heavily depending on the Apps to buy KIN with the dollars and this is still an assumption unless they are contractually bound to buy KIN with the dollar grant.
Raketenernie:  The SEC outcome does play a very crucial role on the future months of kin, I am very pessimistic if kin loses and very optimistic if kin wins. If kin wins and you switch the kre to my demand approach combined with the migration, code and the ads there will be alot of variables pushing demand, we could create a hype will we limiting / stoping and increase of supply. If we continue with the KRE the negative delta of the payouts and the already exiting negative NDB could destroy any upward trend, Just a reminder 300 billiion kin have to be asorbed just to be back at equilibrium of supply and demand-where as 200 million per day is covered up by the ecosystem the rest has to come from speculators, so you see we have to do something vs this unbalance.
Polype01: Very good point and no one can predict what would happen. However, even a loss in the case will not have a huge impact to me on the long run. Why? A loss would impact Kik, not KIN. KIN is not at risk even with a loss. I am 100% comfortable with it. What will happen is that some people will dump their KIN and tank the price (we saw it some weeks ago with the 35% dip when the SEC lawyers only showed their legal document including Kik penalties that they wanted Judge Hellerstein to sign). But then, the path will be cleared and the price will gradually increase. Also, why would people dump their KIN with a 50-60-70% loss? We all assume that people will dump all their KIN if Kik loses but this is not 100% sure. People will still be waiting, especially big bag holders. Also, KIN will not be forbidden to be exchanged/traded/used is Kik loses and it is not like the Telegram case so, as I stated it before, I insist on the fact that a loss is not the end of KIN.

5.	Raketenernie:  You basically spoted that kin is not a true medium of exchange , which you perfectly explained not even in its very own ecosystem, this however would immediately change if had a constant growing value , than beyond the ecosystem it could be used outside in the real world for example for a coffee or a gift card etc... Would you not agree if you had an app and you know every month the kin I hold is worth more, would you not create as much spends as possible in order to get the max of kin inflow as possible, knowing that the kin will be much more worth the next month. If you have buisness sense and want a return then you would do it
Polype01: Yes obviously, if there would be more partners, more Apps and more varied incentives to spend your KIN, there would be more tourism/transfers. For now, why would I bother to move my KIN from one App to another? I am sure that the more partners, the more KIN transfers will happen. So higher KIN value will make KIN moved more from one App to another? Possibly. It might be that a higher KIN value will increase more the movements of KIN from wallets to Exchanges.
Regarding your remark about spends, I am also not sure. Despite the value of KIN during the last past two years, spends haven’t been that developed enough by the developers. Why so? I don’t have the answer and it is a pity. From my perspective, I believe that KIN distributed to the Partners through the KIN KRE should have been done also with the condition of Apps proposing valid amount of spends.
Low number of spends -> lower KIN distributed through the KRE but this is more about fixing the supply and not the demand.
My business sense tells me that it is not black or white. A lot of variables are at hand here and giving dollars as a fix proposal in order to rebalance the disequilibrium between supply and demand will not have a real effect on the long run. 
Raketenernie: You said it yourself the spends where not pushed or not enough, not appealing enough not sustainable enough, why was that? It was the KRE rewarded any activity including earns. So there was no need to focus on spends, especially for the apps (we come to what I wrote earlier) who are only in for the free money and not the tech and idea of kin. 
Polype01: Fully agree
Raketenernie: Cmon we have top tier app still on the ETH blockchain what does it tell us about their true motives using kin, you can tell me what you want they definetly 100 passionate and full in with kin. 
Polype01: I also do not get this. 
Raketenernie: To come back by rewarding dollars for fiat spend on kin, the apps have to create spends and ways for users to buy kin / top up (over direct fiat gateway --> let is be code so we could keep track for the rewards) in order to a) receive a nice junk of the reward and b) by doing so they themselves create real demand pushing the value of kin , meaning that their turnover & profit increases by the same factor kins value has increased. Since it is a joined ecosystem means that all benefit form a value increase of kin
Polype01: I think the KIN KRE algorithm should be refined to ensure to take more into account the SPENDS opportunities and to reward less KIN (reducing the KIN distribution) if relevant SPENDS are not deployed enough. By adjusting those two parameters, the KRE would more balanced. 

6.	Raketenernie: Wrong please read again, I never asked in a single line for a supply reduction or a coin burn, I asked for stop KIN payouts wotsoever and convert them into dollars. Stop spreading false information
Polype01: That’s actually what you did. You explicitly stated in 3/ the following
“By stoping giving them KIN they have a ngeative delta Supply vs Demand, meaning every month their supply of KIN will decrease “ 
Raketenernie: I am not reducing the supply , I am stopping further increase in the supply, meaning the avialiable supply of kin in the market will decrease, the overall circulating supply will continue to be 1.5 trillion, but since more ppl will buy and hold kin it will be more distributed over more user, speculators and apps. 
Polype01: To sum it up: you assume that: reducing the supply, demand stays the same. Price increases. Price increases so more speculators will join and buy KIN and will make the price increasing. The problem to me is the following: we assume that people will come and buy KIN but will people buy just because the price will be greater because we reduce the overall supply?
Raketenernie: This is what we should want. Why do I want to do this as quoted before the supply and demand of kin is not in equilibrium we have a positive delta on the supply of 300 billion kin, which needs to be absorbed by the demand , which is as low of 200 million kin spend per day (in the ecosystem). So it means we have to reduce the oversupply of the 300 billion kin and the least you want to do put even more kin supply on top to make it even bigger. That is why stop supply inflow so you can subtract at least 200 million kin per day of this oversupply. That alone would take 4 years that is why you need to subsidize demands to generate extra demand to absorb faster the 300 billion kin. This is all under the assumption ceteris paribus, if the ecosystem would grow the oversupply could shrink faster or vice versa it could take longer. But we have to tacle the oversupply, it is a hugh oversupply which can kill any kin of upward trend, kill will sit for years on currents value if not lower
Polype01: Due to the unbalance of KIN supply, it will certainly take some time to reach an optimal equilibrium between supply/demand/price. The solution will be reducing supply by putting some extra conditions on Partners to develop spends. Increasing In-App buying for KIN (via Code, Kin Hub…) AND have much more partners joining + marketing. How do we get extra partners? I can think about the Community (and/or KIN Council) contacting any potential partner that could fit KIN’s philosophy in exchange of a grant based on the size of the partner, number of users,….

7.	Raketenernie: Its an equation and eventually they have to reinvest their dollars and buy KIN as they run dry. The point is not only to switch the KRE to dollars but only rewards those who push demand with fiat spend for kin. The so created positiv delta will eventually catch up with the supply and will force the apps to reinvest, which then starts the upward trend in terms of a sustainable increase of kin value and demand. You are right they could aquire as much dollars as possible and exit, but to be fair they can do similar wtih kin aquire as much kin as possible and exit. And to be fair some apps have done so by not even have switched to the kin blockchain. Would you not agree the incentive for an app is greater to stay within the ecosystem and be pro active rewarding them somehing of value instead of an asset which has only 0,13% of its ATH value left. You have to think also a step ahead that as more they focus on the spends and more the denand increases the more the value of kin rises meaning ultimately the econimical benefit using kin will increase for each app, if you want so the econimical benefit of each app using kin right now is only 0.13% what is was once before.
Polype01: No they do not have to. They might leave the Ecosystem as well and we had a case not long ago. Remember Vent leaving all of a sudden due to a Business decision? You assume that partners will reinvest the USD in KIN but you do not have the confirmation they will do it. Here you are not factual and are building your logic based on an assumption.
Some Apps have not switched to KIN3, it is a fact. There should have been a limitation of KIN distributed to those partners by the KRE imo (reducing supply) as they were not 100% migrated to the KIN blockchain (which would have made a tiny increase of demand)
Regarding rewarding Apps that are fostering and promoting spends to make real demand increase, I am 100% in line with you and I agree. But again giving them dollars in parallel can be not sustanaible on the long run. You would reward those Apps in dollars for a limited period of time?
Besides, from a compliance standpoint and based on the transparency document, the KIN budget is defined in KIN only. Rewarding dollars to partners via the KRE is not mentioned.
Raketenernie: I answered before here my view, these are not true partners and we should not support them, one thing which could engage apps to stay and especially hold kin is add to my approach staking but also in dollars. I thought about it there is so much liquduity it will take a very long time that demand can overcome the 300 billion kin , unless 300 of us buy 1 billion kin each may be an idea. Staking could help that apps not dump all and cash out and do a runner, have a second incentive to stay, hold and acquire kin.
Polype01: These are partners. They joined probably as “pilot partners” in exchange of a payment and also probably got another payment for maintenance costs for being an Ecosystem node. 
To not support them is a bit harsh but they should be pushed/incentivized to do more than they are doing. They are in the Ecosystem so even if they could do more, they are true partners. I proposed staking/periodic interest in a GitHub and it is not as easy as it seems because it could be bad perceived by the SEC. I believe that if KIN is a true currency, it should be later possible to get an interest as it is the case with real FIAT that is on a bank account; If there is an interest, then THIS would be an incentive to buy and hold KIN. The more KIN you hold, the more interest you will receive. But this is another topic.

8. Raketenernie: Yes I certainly believe it will help to create more adoption and kin awareness. It is not crazy to believe that distributing FIAT could bring new apps to the ecosystem. Especially new apps are the one who need to rebuy KIN which would you instanly create a function economical circle. I cannot speak about other projects who might be more suitable for whatever reason. May be others reward more, but KIN does have a funtiong network with cross joined apps , which is appealing to medium seized apps.
Polype01: Distributing FIAT might attract people and developers. But what will attract much more partners is the image, reputation, potential, adoption, vision of the project. Price would attract speculators. If KIN has no speculative demand now is mostly because of the lack of exchanges and the SEC v. Kik case. Also a lack of marketing due to the fact that we need all the SEC Cloud dissipated and all technical aspects tackled first.
Fundamentals. If fundamentals are good, it will attract naturally people and partners. The SEC v Kik case finished, a lot of things will move forward. Solana mainnet golive will probably be done at the same time most of the KIN Ecosystem will be migrated on the Solana Network.
Raketenernie: You are right but to be fair kins reputation is that of a shit coin, as much I love this project its reputation is burned. Speculator were send to hell and they are gone but the truth is you need their capital to push a project, kin approach of focusing only on users was a false strategy it came a decade to early. In the infant stage of an advancing technology you need a lot of capital, best example is Tesla, look how much capital they collected and not even had a finised product to be produced on mass. I blieve if SEC turns out good, solona can do what is promised code looks fine there will positive momentum, but you still have strong opponents mr hyperinflation and the oversupply.Especially the oversupply will swallow up alot of the positive momentum.
Polype01: Since the beginning I am reading you, I have noticed that you are really harsh on the project and I still do not know the reason. Naming KIN as “Shitcoin” is really exaggerated. You are naming KIN as such because of its low price. Would KIN be still a “shitcoin” if it were in Top 100 in CMC?
If speculators are sent to hell, well it is their fault and they are to blame. KF owes nothing to people who overinvested and lost money. Crypto is highly risky and some people did overcommit and became mean haters. KIN is still backed by some Funds and original investors from the first part of the TDE and I believe all the KIN supporters also have quite some bags.
You say it yourself. Tesla has billions dollars valuations and still has an “unfinished product”. Same for KIN with 100M dollars collected. I believe everything that has been on hold will be released and as soon as we have a decision, the speculative demand will increase, much more than the real demand. Taking my report into consideration, we need roughly the demand to be higher than 13b KIN /month (roughly 80k USD at today price) to have demand superior than the supply. This would mean 80000 users buying 1 dollar of KIN in-App in code/KinHub. This seems not to be impossible to reach, provided we have spends of course. If speculative demand comes also, the number of users buying in KIN could be smaller. So the current state of demand is a bit of a bummer now but I am sure if everything is deployed well, demand will be fixed in the midterm

9. Raketenernie: Wrong it is not a reduction it is setting the supply fixed for now until we meet equilibrium
Polype01: You are playing with words here.
Stopping the KIN supply by stopping the KIN distributed through the KRE is a reduction of supply overtime. If you stop distributing KIN via the KRE during the next period, about 3b KIN will not be distributed. So it IS a reduction of KIN supply. 
Raketenernie: Its a delay , eventually once we are back in equilibrium kin can be granted again, accordingly the resilence of the ecosystem.
Polype01: Yes but then, your dollar incentive once stopped and we go back to the KRE payout with KIN, then the billions of KIN redistributed will also tank the price. There will be also a shock on the supply/demand equilibrium when your dollar incentive will stop

10. Raketenernie: This is your personal view based on what facts? I provided you a concept both proven by history and econimcs. Demand will never ever happen in the current state, as again the incentive to buy , hold or earn kin is so little, as a direct result of the hyperinflation. the only way of stoping hyperinflation is puting pressure on the supply and increase the value of kin which then will bring the above asked demand combined with your oganical growth the ecosystem will eventually come to an equlibrium where rewards as such wont be neccessary.
Polype01: Not quite. Demand is happening. Kin Ads will be deployed more and more. That’s concrete AND sustainable for the Ecosystem. The proposal for paying Exchanges, if adopted, would also be a contribution towards demand. Demand is happening. Again, not black or white. Code Wallet to-be deployed 3Q2020…
Reducing hyperinflation could be done with the following: Adjusting the KIN KRE distribution and put as a parameter in the formula the development of spends in App as a condition -> limit the distribution of KIN if the top Apps do not adapt (reducing suppl and the positive consequence,. if Apps still want to receive more KIN, they would need to develop more spends. 
Raketenernie: Look I bet you the top tier 5 apps having billions of kin, do need not more kin, they would be happy to receive dollars. To receive dollars they have to push spends sounds to me good. Kin Ads, we do not know how significant that will be, I fear it will be as mighty as several thousands dollars of fiat spend for kin per month, a metric for example KF hardly ever published as it is so significant small that is shows the illness of the ecosystem. PAying exchanges to list kin , to get on binance and bitfinex and coinbase hm may be I can agree on that but that brings nothing sustainable to the table , rather I would like to see staking or that apps list kin cuz they like kin not because we paid us, cuz these are exchanges who will drop kin as soon there is nothing in for them.
Polype01: The top App, from what I saw do NOT have that much KIN when I checked individually the KIN KRE Addresses they submitted. Most of all top10Apps are dumping KIN on Exchanges directly (I made that check some weeks ago so maybe it has changed in between). So it is not totally true to assume that the still have billions and billions of KIN in reserve.
From my report, several thousands of dollars/month (let’s say 10000USD) means that this is a demand increase equal to 12,5% per month (as 80000USD = about 13b KIN = 1 month) so it is quite significant and not totally irrelevant. I am also curious to see the exact numbers from Kin Ads and hope that Josh will communicate a bit about it later.
You have to admit that not communicating financial/strategic numbers can be understood and even with all my curiosity, we cannot force KF to disclose those numbers. This is no company stock and the KF, as a non-profit organization, is not forced to publish those numbers. Deciding not to show those numbers does not mean that they are ill, shady or in financial difficulties. Do you know other foundations publicly disclosing those numbers?
Deciding to pay/give bonus/incentive to list KIN can be nice but it is also increasing the supply. I think that this might be interesting if limited in time. But here, you see, we are talking about paying in KIN and not in USD. So this is better for the KIN philosophy but we need to ensure that the cost is lower than the benefits. In other words, we need to ensure that the payment in KIN to that exchange will give a higher return in KIN (people buying KIN that is higher than the grant given to the Exchange).

11. Raketenernie: As said in the proposal per month we reward 200.000 dollars distributed to apps accordingly to their share of the 3 presented variables where fiat spend for kin is the most important. It is a subzidized ecosystem focusing only on the demand leading to an increase of value of kin. It is not artificial as the increase of value comes from within the ecosystem from its very own demand. Combined with a change of incentive to hold kin , user, speculator and devs will increase their demand to acquire a portion x of KIN. It is exactly the oposite what the KRE does now , instead of subzidizing liquidity we subsidize true demand.
Polype01: You can write it in multiple different ways, it does not matter. This would work only if the Ecosystem Apps will willingly and freely buyback KIN with the received dollars. If they HAVE TO buy KIN contractually with the received FIAT (or even part of it) then it would be different and I could consider the idea further. But then again, what happens if KF is running out of money?
I was advocating for staking system/interest rate based on the KIN stash that any user would hold for a specific amount of time on a specific address but this could have been interpreted negatively by the SEC and hence, was not discussed further.
Who knows, one day, if KIN is officially recognized as a currency, then we might see people buying more KIN in order to hold more KIN and in order to receive an interest based on their KIN stash. 
Raketenernie: As said staking is not a bad idea but combined with the adjustment on the KRE. Getting interest on kin does not make it a security, money give you interest /return too. I just had an idea while you were saying why would apps willingly and freely buyback kin, cuz a) they run dry and they have to buy or b) we give them additional incentive in form of a reward (meaning they gain a abitrage advantage)--> extra profit for them.
Polype01: True. Receiving an interest is a basic function of money and should be, imo, possible but in those uncertain times, with the SEC case, we do not want to have that uncertainty on top of what we have; What happens if – worst case scenario – SEC decides that KIN is a security? What would happen for the people who received KIN from staking? Staking can be considered later maybe later and is not a priority but I believe that it would help increasing demand.
Raketenernie: I repeat it is not about the price of kin, it is about abolishing hyperinflation and rewarding demand and increasing demand and yes this means kins price should not be 0,13% what it used to be. I bet all my kin, if KIN would be close to ATH price levels, the ecosystem and the adoption of kin would be at least 10x bigger. Each app, user, speculator would have had a 93x higher incentive in terms of value to hold buy or use kin, just think about 93x times. What do you or anyone here do not understand , it is plane simple right there.
Polype01: Higher price as an incentive to attract more investors is obvious. I can imagine an investor willingly investing in a top100 coin instead of investing in a top 500 coin to mitigate the risks. But price will be the positive consequence on growing organic demand from the Apps.
I understand what you mean. Your proposal to boost demand has the merit to exist and might be considered by KF.
Raketenernie: I am not saying my proposal is pefect or it will work they way I beleive, but it should be clear for everyone, we have oversupply and hyperinflation and this does not encourage, to use, hold or buy kin. We have to create incentives fixing this core problem.
1.	Look at the variables we reward
2.	Look what currency we reward and the amount
3.	Look who we reward and what is their true value for the ecosystem
4.	Look for ways to bring demand and supply back to equlibrium
5.	Look who and how we can bring econimcal measurable value to kin
I will adjust my proposal when I have time I will add a FAQ section. By the end of the day if we start reward true demand that would be a start even tho I strongly beleive subsidizing demand will do a lot, a lot more than paying exchanges , doing a direct buy back etc... as we would subsidize the ecosystem.

Polype01: Rewarding true demand and rewarding Apps that develop relevant, useful spends would be nice and I agree with you on this. Pushing some Apps by developing KIN Spends could be nice by, just an idea popping up now, paying those Apps with KIN but those KIN would be blocked for any movement on a separate account for some years?
